fem/fe/fe_nd.cpp

```diff
@@ -314,7 +314,6 @@ void ND_HexahedronElement::CalcVShape(const IntegrationPoint &ip,
 #ifdef MFEM_THREAD_SAFE
    Vector shape_cx(p + 1), shape_ox(p), shape_cy(p + 1), shape_oy(p);
    Vector shape_cz(p + 1), shape_oz(p);
-   Vector dshape_cx(p + 1), dshape_cy(p + 1), dshape_cz(p + 1);
```

Comment:
Are these not necessary for thread safety?

Reply:
They are defined in the scope of the `if` statement just below